File: 🟢 Easy/3162 - Find the Number of Good Pairs I/_3162FindtheNumberofGoodPairsI.py
class Solution:
    def numberOfPairs(self, nums1: List[int], nums2: List[int], k: int) -> int:
        count = 0
        for i in range(0, len(nums1)):
            for j in range(0, len(nums2)):
                if nums1[i] % (nums2[j] * k) == 0:
                    count += 1
                else:
                    continue

        return count

File: 🟢 Easy/3162 - Find the Number of Good Pairs I/_3162FindtheNumberofGoodPairsI.c
int numberOfPairs(int* nums1, int nums1Size, int* nums2, int nums2Size, int k) {
    int count = 0;
    for (int i = 0; i < nums1Size; i++) {
        for (int j = 0; j < nums2Size; j++) {
            if (nums1[i] % (nums2[j] * k) == 0) {
                count++;
            } else {
                continue;
            }
        }
    }
    return count;
}